src/libxively/control_topic/xi_control_topic_layer.c

```diff
@@ -196,6 +196,7 @@ xi_state_t xi_on_control_message( xi_context_handle_t in_context_handle,
                 xi_sft_on_connected( layer_data->sft_context );
 #endif
             }
+            break;
         }
```

Comment:
wondering why lack of this break did not make trouble: a "fake" empty or trash message eaten up by the SFT module.

Reply:
I think because the cbor decode failed on litter, returned NULL `control_message` which is mapped to no-operation in the `xi_sft_on_message`.

Reply:
Yes, I had the same concern.  I was hoping that you would know. :)